src/Explorer/Notebook/useNotebook.ts

```diff
@@ -173,8 +175,10 @@ export const useNotebook: UseStore<NotebookState> = create((set, get) => ({
     isGithubTree ? set({ gitHubNotebooksContentRoot: root }) : set({ myNotebooksContentRoot: root });
   },
   initializeNotebooksTree: async (notebookManager: NotebookManager): Promise<void> => {
+    const notebookFolderName = userContext.features.phoenix === true ? "My Notebooks Scratch" : "My Notebooks";
```

Comment:
dont you have a util method for this

Reply:
useNotebook itself is kind of util and global store. So i think this should be good.